ci-operator/config/jboss-fuse/fuse-xpaas-qe-container/jboss-fuse-fuse-xpaas-qe-container-main__jboss-fuse-ocp4.14-lp-interop.yaml

```
@@ -0,0 +1,53 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+  test-image:
+    name: xpaas-qe
+    namespace: ci
+    tag: 7.12.x-openjdk1.8_local
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: jboss-fuse-interop-aws
+  cron: 0 6 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
+          {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
+          {"step": "gather-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
+          {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
+          {"step": "jboss-fuse-*", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "FUSEQE", "jira_epic": "FUSEQE-19087"}
```

Our team removed the `jira_epic` for tickets in `LPTOCPCI` project in the past for all [scenarios](https://github.com/openshift/release/pull/41867), can you please make these changes
```suggestion
          {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI"},
          {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "jira_project": "LPTOCPCI"},
          {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
          {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI"},
          {"step": "jboss-fuse-*", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "FUSEQE", "jira_epic": "FUSEQE-19087"}
```

---

Sure